u2f-ref-code/java/src/com/google/u2f/key/messages/TransferAccessResponse.java

```diff
@@ -0,0 +1,104 @@
+package com.google.u2f.key.messages;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+import com.google.common.base.Preconditions;
+import com.google.u2f.server.ControlFlags;
+
+/**
+ * TransferAccessResponse is constructed by a device to whom access has been transferred. It wraps 
+ * the TransferAccessMessage chain with necessary information to register a new key.
+ */
+public class TransferAccessResponse extends AuthenticateResponse {
+  private final byte controlFlags;
```

Comment:
if some of these exist in AuthenticateResponse, you don't want to re-state them here as that can lead to weird bugs.